src/client/app/reducers/readings.js

```diff
@@ -16,7 +16,12 @@ import * as readingsActions from '../actions/readings';
  * @type {State~Readings}
  */
 const defaultState = {
```

Comment:
In that case you can always keep `actions/readings.js` and just add `actions/barChartReadings.js`, etc. as necessary.